### PR TITLE
EID-1542: Make the no-js "Continue" button the same as in the Hub

### PR DESCRIPTION
**Pull request by CharlesIC**
The Hub button says "Continue", not "Submit".
Also clean up the unnecessary args as the button text is no longer customisable.

### CODE REVIEW

**Review by JohnWattsGDS — APPROVED** [review]
This should be ~submitted~ continued with.

**Review by tommrr — APPROVED** [review]

**Comment by CharlesIC** [comment]
Your reviews don't count @tomrosier - it's a grey tick 😕

File: proxy-node-shared/src/main/java/uk/gov/ida/notification/SamlFormViewBuilder.java [hunk 1]
```diff
@@ -10,13 +10,13 @@
...
-    public SamlFormView buildRequest(String url, AuthnRequest authnRequest, String submitText, String relayState) {
+    public SamlFormView buildRequest(String url, AuthnRequest authnRequest, String relayState) {
         String samlMessage = marshaller.transformToString(authnRequest);
         String encodedSamlMessage = Base64.encodeAsString(samlMessage);
-        return buildRequest(url, encodedSamlMessage, submitText, relayState);
+        return buildRequest(url, encodedSamlMessage, relayState);
     }
 
-    public SamlFormView buildRequest(String url, String encodedSamlMessage, String submitText, String relayState) {
+    public SamlFormView buildRequest(String url, String encodedSamlMessage, String relayState) {
```
Comment (instantiator) [inline proxy-node-shared/src/main/java/uk/gov/ida/notification/SamlFormViewBuilder.java]: It looks like `submitText` wasn't used in this method anyway?

File: proxy-node-gateway/src/test/java/uk/gov/ida/notification/apprule/SamlResponseExceptionMapperResourceRuleTest.java [hunk 2]
```diff
@@ -44,10 +44,9 @@ public void before() {
                 new SamlFormView("postUrl",
                                  "samlMessageType",
                                  "encodedSamlMessage",
-                                 "submitText"));
+                                 "relayState"));
```
Comment (instantiator) [inline proxy-node-gateway/src/test/java/uk/gov/ida/notification/apprule/SamlResponseExceptionMapperResourceRuleTest.java]: This is interesting - it looks like you've fixed a bug. (My understanding was that the `SamlFormView` constructor would have dropped from 5 params to 4 params (removing the `submitText`), but this went from 4 to 4...

**Review by instantiator — APPROVED** [review]

File: proxy-node-shared/src/main/java/uk/gov/ida/notification/SamlFormViewBuilder.java (hunk 1 above)
Comment (CharlesIC) [reply proxy-node-shared/src/main/java/uk/gov/ida/notification/SamlFormViewBuilder.java]: Yep - this is just cleaning things up

File: proxy-node-gateway/src/test/java/uk/gov/ida/notification/apprule/SamlResponseExceptionMapperResourceRuleTest.java (hunk 2 above)
Comment (CharlesIC) [reply proxy-node-gateway/src/test/java/uk/gov/ida/notification/apprule/SamlResponseExceptionMapperResourceRuleTest.java]: Yep - there were some changes made a while ago to the SAML form view and not everything was cleaned up properly then

**Pull request merged**